data/formats-data.js

```diff
@@ -5229,21 +5229,21 @@ exports.BattleFormatsData = {
 		tier: "RU"
 	},
 	pumpkaboo: {
-		randomBattleMoves: ["willowisp","shadowsneak","fireblast","synthesis","seedbomb"],
+		randomBattleMoves: ["willowisp","shadowsneak","destinybond","synthesis","bulletseed","leechseed"],
 		tier: "LC"
 	},
 	pumpkaboosmall: {
-		randomBattleMoves: ["willowisp","shadowsneak","fireblast","synthesis","seedbomb"],
+		randomBattleMoves: ["willowisp","shadowsneak","destinybond","synthesis","bulletseed"],
 		unreleasedHidden: true,
 		tier: "LC"
 	},
 	pumpkaboolarge: {
-		randomBattleMoves: ["willowisp","shadowsneak","fireblast","synthesis","seedbomb"],
+		randomBattleMoves: ["willowisp","shadowsneak","leechseed","synthesis","bulletseed"],
 		unreleasedHidden: true,
 		tier: "LC"
 	},
 	pumpkaboosuper: {
-		randomBattleMoves: ["willowisp","shadowsneak","fireblast","synthesis","seedbomb"],
+		randomBattleMoves: ["willowisp","shadowsneak","leechseed","synthesis","seedbomb"],
```

Comment:
I forgot.

In all honesty though, maybe they should all keep Seed Bomb. LC damage calcs mean that Bullet Seed outdamages Seed Bomb 67% of the time, while at higher levels, the reverse is true.